components/split-button/index.tsx

```diff
@@ -9,113 +9,38 @@ import Overlay from '../overlay';
 import Menu from '../menu';
 import ConfigProvider from '../config-provider';
 import { dom, obj, func } from '../util';
+import type { SplitButtonProps } from './types';
+import type { Item as MenuItem } from '../menu';
```

Comment:
Eslint:  import is duplicated
e.g. import Menu, { type Item as MenuItem } from '../menu'